packages/ove-asset-manager/src/OVE.Service.AssetManager/Views/OVEAssetModel/Edit.cshtml

```diff
@@ -0,0 +1,80 @@
+﻿@model OVE.Service.AssetManager.Models.OVEAssetModel
+@using OVE.Service.AssetManager.Domain
+@inject ServiceRepository Services
+
+@{
+    ViewData["Title"] = "Edit";
+}
+
+<h2>Edit</h2>
+
+<h4>ImageFileModel</h4>
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        <form asp-action="Edit" enctype="multipart/form-data">
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <input type="hidden" asp-for="Id" />
+            <div class="form-group">
+                <label asp-for="Project" class="control-label"></label>
+                <input asp-for="Project" class="form-control" />
+                <span asp-validation-for="Project" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="Name" class="control-label"></label>
+                <input asp-for="Name" class="form-control" />
+                <span asp-validation-for="Name" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="Description" class="control-label"></label>
+                <input asp-for="Description" class="form-control" />
+                <span asp-validation-for="Description" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="Service" class="control-label"></label>
+                @Html.DropDownList("Service", Services.GetServices(), new { @class = "form-control" })
+                <span asp-validation-for="Service" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="AssetMeta" class="control-label"></label>
+                <input asp-for="AssetMeta" class="form-control" />
+                <span asp-validation-for="AssetMeta" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="StorageLocation" class="control-label"></label>

```

Comment:
The Storage location should not be a form field, since a) it is read only, b) it should not be user changeable

In addition, if it is the actual location, this is a worthless field, since the unauthenticated user would have no access to a protected object store, so only a summary location would be required

Reply:
This feild is set to be readonly on line 45.

Reply:
I think, as per other queries, this might be better as hidden, rather than just read-only?